explainer/index.bs

```diff
@@ -23,19 +23,59 @@ Issue(tabatkins/bikeshed#2006): Set up cross-linking into the WebGPU and WGSL sp
 Issue(gpuweb/gpuweb#1321): Complete the planned sections.
 
 
-# Motivation # {#motivation}
+# Introduction # {#introduction}
 
-See [Introduction](https://gpuweb.github.io/gpuweb/#intro).
+WebGPU is a proposed Web API to enable webpages to use the system's [GPU (Graphics Processing Unit)](https://en.wikipedia.org/wiki/Graphics_processing_unit) to perform computations and draw complex images that can be presented inside the page.
+This goal is similar to the [WebGL](https://www.khronos.org/webgl/) family of API, but WebGPU enables access to more advanced features of GPUs.
+Whereas WebGL is mostly for drawing images but can be repurposed (with great effort) to do other kinds of computations, WebGPU has first-class support for performing general computations on the GPU.
 
-Issue: This section hasn't been written;
-[more actual motivation](https://w3ctag.github.io/explainers)
-is needed (here, in the spec Introduction section, or both).
+## Use cases ## {#use-cases}
 
+Example use cases for WebGPU that aren't addressed by WebGL 2 are:
+
+ - Drawing images with highly-detailed scenes with many different objects (such as CAD models). WebGPU's drawing commands are individually cheaper than WebGL's.
+ - Execute advanced algorithms for drawing realistic scenes. Many modern rendering techniques and optimizations cannot execute on WebGL 2 due to the lack of support for general computations.
+ - Execute machine learning models efficiently on the GPU. WebGL can be massaged to execute machine learning but it is difficult and sub-optimal.
+ - Provide a foundation to expose modern GPU functionality on the Web. WebGPU exposes functionality that became available in native around 2010 but is structured similarly to all current native GPU APIs. There are plans to later extend it to have more modern functionality. See also: [[#why-not-webgl3]].
+
+Concrete examples are:
+
+ - Improving existing Javascript 3D libraries like Babylon.js and Three.js with new rendering techniques (compute-based particles, fancier post-processing, ...) and offloading on the GPU expensive computations currently done one the CPU (culling, skinned model transformation, ...)
+ - Porting newer game engines to the Web, and enable engines to expose more advanced rendering features. For example Unity's WebGL export uses the "mobile" feature set, but WebGPU could use a higher feature set.
```

Comment:
Not sure I understand the bit about Unity's WebGL. If it can't target some mobile GPUs because of their limits of features, how is WebGPU going to help this case?

Reply:
Rephrased to use "lowest" instead.